plugin-maven/src/main/java/com/diffplug/spotless/maven/AbstractSpotlessMojo.java

```diff
@@ -373,9 +373,9 @@ private UpToDateChecker createUpToDateChecker(Iterable<Formatter> formatters) {
 		}
 		final UpToDateChecker checker;
 		if (upToDateChecking != null && upToDateChecking.isEnabled()) {
-			getLog().info("Up-to-date checking enabled");
 			checker = UpToDateChecker.forProject(project, indexFile, formatters, getLog());
 		} else {
+			getLog().info("Up-to-date checking disabled");
```

Comment:
Changed logging slightly because disabled up-to-date checking will be an uncommon path that requires explicit configuration. Thus, the plugin should log something on such an uncommon path